### PR TITLE
Fix when playback recording with status >400 throws an error

### PR DESCRIPTION
**Pull request by ei0haro**
Fix when playback recording with status >400 throws an error

### CODE REVIEW

**Review by nordstrand — APPROVED** [review]

File: troxy-server/src/main/java/no/sb1/troxy/util/SimulatorHandler.java [hunk 1]
```diff
@@ -232,9 +232,7 @@ public void handle(String target, org.eclipse.jetty.server.Request jettyRequest,
         byte[] contentBytes = response.getContent().getBytes(response.discoverCharset());
         /* status */
         try {
-            int code = Integer.parseInt(response.getCode());
-            if (code < 400) servletResponse.setStatus(code);
-            else servletResponse.sendError(code, response.getReason());
```
Comment (aregr) [inline troxy-server/src/main/java/no/sb1/troxy/util/SimulatorHandler.java:237]: Foreslår heller at servletResponse.sendError(code, response.getReason()) endres til 

servletResponse.setStatus(code, response.getReason());

Comment (aregr) [reply troxy-server/src/main/java/no/sb1/troxy/util/SimulatorHandler.java:237]: Suggest we use this fix instead:

https://github.com/SpareBank1/Troxy/pull/27

**Review by aregr — APPROVED** [review]

File: troxy-server/src/main/java/no/sb1/troxy/util/SimulatorHandler.java (hunk 1 above)
Comment (ei0haro) [reply troxy-server/src/main/java/no/sb1/troxy/util/SimulatorHandler.java:237]: Då denna fixas i andra pullrequester så tar jag bort denna

**Comment by ei0haro** [comment]
Fix in other branch

**Pull request closed without merging**